src/js/msp.js

```diff
@@ -311,6 +314,13 @@ var MSP = {
         return bufferOut;
     },
     send_message: function (code, data, callback_sent, callback_msp, doCallbackOnError) {
+        if(CONFIGURATOR.virtualMode){
```

Comment:
And more (not going through the whole list). Just minor issue, but we like the code to be consistent.